src/amo/components/Search/index.js

```diff
@@ -60,49 +77,66 @@ export class SearchBase extends React.Component {
       LinkComponent,
       count,
       enableSearchSort,
+      errorHandler,
       filters,
-      hasSearchParams,
       loading,
-      page,
+      paginationQueryParams,
```

Comment:
My bad, the paginator itself doesn't do that as it's too generic, but the search component will. So for certain searches we just use the filters, but here we specify which ones to include.